nginx.conf

```diff
@@ -88,14 +88,16 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/arbitrum-faq/$                                $scheme://$http_host/docs/$1/arbitrum/arbitrum-faq/                         permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/migrating-a-subgraph/$               $scheme://$http_host/docs/$1/cookbook/upgrading-a-subgraph/                 permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/quick-start/$                        $scheme://$http_host/docs/$1/quick-start/                                   permanent;
-        rewrite ^/docs/en/substreams/(?!index\.).+$                                     https://substreams.streamingfast.io                                         permanent;
-        rewrite ^/docs/en/firehose/(?!index\.).+$                                       https://firehose.streamingfast.io                                           permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/developer/assemblyscript-api/$                $scheme://$http_host/docs/$1/developing/graph-ts/api/                       permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/developing/assemblyscript-api/$               $scheme://$http_host/docs/$1/developing/graph-ts/api/                       permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/deprecating-a-subgraph/$             $scheme://$http_host/docs/$1/managing/transfer-and-deprecate-a-subgraph/    permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transferring-subgraph-ownership/$    $scheme://$http_host/docs/$1/managing/transfer-and-deprecate-a-subgraph/    permanent;
```

Comment:
These are the redirects that should have been included in #741; my bad for not noticing in my review. :)